docs/modules/data-structures/pages/updating-map-entries.adoc

```diff
@@ -173,9 +174,10 @@ Go::
 -- 
 [source,go]
 ----
-countries.Put( 1, "Korea" ); <1>
-countries.Put( 2, "France", "Spain" ); <2>
-countries.Put( 3, "United States", "Canada" ); <3>
+ctx := context.Background()
+countries.Replace(ctx, 1, "Korea") <1>
```

Comment:
```suggestion
result, _ := countries.Replace(ctx, 1, "Korea") <1>
```
How about prepending these lines with return value and a ignored error? And also include a comment that errors are ignored for brevity?